runtime/src/bank.rs

```diff
@@ -2015,6 +2022,14 @@ impl Bank {
         let mut tx_count: u64 = 0;
         let err_count = &mut error_counters.total;
         for ((r, _hash_age_kind), tx) in executed.iter().zip(txs.iter()) {
+            if let Some(debug_keys) = &self.transaction_debug_keys {
+                for key in &tx.message.account_keys {
+                    if debug_keys.contains(key) {
+                        info!("result: {:?} tx: {:?}", r, tx);
```

Comment:
should we also log which slot, this is, i.e. `self.slot`

Reply:
I thought we had prints which tell us which slot is processing.

Reply:
yeah, but it can be hard sometimes to figure out which slot is being replayed when you have parts of multiple slots being processed at the same time (don't get the "frozen" separator between them)

Reply:
ok, added